science/limesuite/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           cxx11 1.1
+PortGroup           github 1.0
+PortGroup           wxWidgets 1.0
+
+name                limesuite
+version             v19.01.0
```

Comment:
Reviewing the GIT repo for this project, it is fairly active development ... so, I'd recommend having a "release" version as well as a "devel" version, much like I do with GNU Radio & other active projects that I maintain. It's easy to add a subport for devel & then tweak any release or devel specific needs within that subport's bounds.

Reply:
I did. Please check if it is correct. I used your OpenBLAS as main template. Just one question: devel subport could track HEAD instead of a commit that we need to change? thanks

Reply:
Our (MP) general policy is to track a specific archive, release, or commit unless it is impossible to do so. Doing so makes debugging issues easier. OpenBLAS is a fine example, as would be GR or GNSS-SDR or libuv or many others.